.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tempesta-tech.com"]
+	path = tempesta-tech.com
+	url = https://github.com/tempesta-tech/tempesta-tech.com.git
```

Comment:
Here we should use git@github.com:tempesta-tech/tempesta-tech.com.git (for ssh cloning) instead of https